src/ansys/geometry/core/designer/body.py

```diff
@@ -962,9 +971,9 @@ def set_fill_style(  # noqa: D102
     @protect_grpc
     @check_input_types
     @min_backend_version(25, 1, 0)
-    def set_color(self, color: Union[str, Tuple[float, float, float], mcolors.Colormap]) -> None:
```

Comment:
I meant to "use" mcolors to extract the hex. Nonetheless, it doesn't make sense to pass a Colormap @umutsoysalansys. You use colormap to extract the hex value or rgb(a) value of a color. But you have to understand Colormap as a "dictionary". We are only expecting a color - a single color. That's why accepting the color as a string, HEX or as RGB(A) is enough. And you use colormap to convert any of them to HEX - which is what you were already doing...

Bottom line, I just fixed your type hint because it didn't make sense. internally it was fine.

Reply:
PR is ready but as long as the unstable image is not passing we can't verify it... @smereu and @b-matteo are aware of this. We need to fix that first. We can't merge it otherwise.

Reply:
That's because the null value is not implemented on the ``windows-latest`` image but on the ``windows-latest-unstable``... however, that image is currently broken. That's why we have to wait for the team to fix the image first.